modules/server/templates/scripts/forseti-server/run_forseti.sh.tpl

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /home/ubuntu/forseti_env.sh
+
+
+# set -x enables a mode of the shell where all executed commands are printed to the terminal.
+# With this  enabled, we should not put anything private/secret in the commands called because
+# they will be logged.
+set -x
+
+# Remove any CAI temp files generated by the forseti process.
+# Failed inventory due to various reasons could cause the temp files not being deleted.
+sudo rm -rf /tmp/forseti-cai-*
+
+# Put the config files in place.
+sudo gsutil cp gs://${storage_bucket_name}/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
```

Comment:
After the change is made above to the main.tf, these variables will need to be replaced from the TF var syntax to the environment var syntax. For example, instead of `${storage_bucket_name}` put `$SCANNER_BUCKET`, which is the env var that gets exported by the forseti_env.sh script.

Reply:
Done